sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/CosmosPartitionPlannerITest.scala

```diff
@@ -185,6 +195,60 @@ class CosmosPartitionPlannerITest
       Some(90))
   }
 
+  "createInitialOffset" should "only ever retrieve one page of changes per feedRange" in {
+
+    val container = this.cosmosClient
+      .getDatabase(containerConfig.database)
+      .getContainer(containerConfig.container)
```

Comment:
I think I missed. that you can actually use the container from `CosmosContainer` trait :D

Reply:
please ignore what you have is correct. ignore this.